packages/testnet-faucets/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM node
 
 WORKDIR /src
 
+RUN apt-get -qq update && apt-get install -y libhidapi-dev libusb-1.0-0-dev
+ENV CIRCLECI=true
```

Comment:
Is this because of the `node_hid` issue?

Reply:
yep, added a comment for this.